src/include/daos/rpc.h

```diff
@@ -52,6 +52,7 @@ enum daos_module_id {
 	DAOS_SEC_MODULE		= 9, /** security framework */
 	DAOS_DTX_MODULE		= 10, /** DTX */
 	DAOS_MAX_MODULE		= 64, /** Size of uint64_t see dmg profile */
+	DAOS_NR_MODULE		= 11  /** number of defined modules */
```

Comment:
Would be better to place DAOS_NR_MODULE between DAOS_DTX_MODULE and DAOS_MAX_MODULE.

Reply:
Fixed in #6200